docs/_sass/minima/_base.scss

```diff
@@ -288,7 +288,7 @@ table {
     text-align: center;
   }
   .site-header:before {
-    content: "AB-3";
+    content: "TutorAid";
```

Comment:
Wonder what this change does? A little concerned about merging because I don't know what it's for

Reply:
Not really sure what this affects too. I have changed it back.